### PR TITLE
chore: regenerate entity client spec and types

### PR DESCRIPTION
**Pull request by morzhan**

### CODE REVIEW

File: clients/entity-client/src/openapi-runtime.json [hunk 1]
```diff
@@ -853,10 +706,79 @@
       }
     }
   },
-  "components": {},
-  "servers": [
-    {
-      "url": "https://entity.sls.epilot.io"
```
Comment (jpinho) [inline clients/entity-client/src/openapi-runtime.json:859]: I think we want to keep this!

File: clients/entity-client/package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "test": "jest",
     "typescript": "tsc",
     "bundle-definition": "webpack",
-    "openapi": "node ../../scripts/update-openapi.js https://docs.api.epilot.io/entity.yaml",
+    "openapi": "node ../../scripts/update-openapi.js /Users/hansweber/Developer/entity-api/lambda/ApiHandlerFunction/openapi.yml",
```
Comment (jpinho) [inline clients/entity-client/package.json]: Revert 🙂

**Review by jpinho — CHANGES_REQUESTED** [review]
LGTM! Just some things to be fixed, other than that looks Hansome!

File: clients/entity-client/package.json (hunk 2 above)
Comment (morzhan) [reply clients/entity-client/package.json]: done in second push!

**Review by jpinho — APPROVED** [review]

**Pull request merged**